### PR TITLE
Css frameworks

### PR DESCRIPTION
**Pull request by JeanArce**
Ready for review

### CODE REVIEW

File: src/scss/index.scss [hunk 1]
```diff
@@ -1 +1,131 @@
+@import 'variables';
+
+$dark: #eab5c0;
```
Comment (Fermain) [inline src/scss/index.scss]: How come this doesn't live in _variables?

File: src/scss/index.scss [hunk 2]
```diff
@@ -1 +1,131 @@
...
+    background-color: $dark
+}
+
+.login-container {
+
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (Fermain) [inline src/scss/index.scss]: There are bootstrap classes for this

File: src/scss/index.scss [hunk 3]
```diff
@@ -1 +1,131 @@
...
+
+
+    @media (max-width: 767px) {
+        height: auto;
```
Comment (Fermain) [inline src/scss/index.scss]: I think you can do this with sizing utilities, like `h-auto h-md-100vh`

File: src/scss/index.scss [hunk 4]
```diff
@@ -1 +1,131 @@
...
+    }
+
+    .login-inner {
+        width: 600px;
+       
+        background-color: #FFFFFF;
```
Comment (Fermain) [inline src/scss/index.scss]: `bg-white` or `bg-light`

File: src/scss/index.scss [hunk 5]
```diff
@@ -1 +1,131 @@
...
+        border-radius: 10px;
```
Comment (Fermain) [inline src/scss/index.scss]: https://getbootstrap.com/docs/5.3/utilities/borders/

File: src/scss/index.scss [hunk 6]
```diff
@@ -1 +1,131 @@
...
+            
+
+            @media (min-width: 768px) {
+               @include border-left-radius(10px);
+                width: 100%;
+                height: 100%;
+              
+            }
+
+            width: 100%;
```
Comment (Fermain) [inline src/scss/index.scss]: `w-100` `w-md-100` etc

File: src/scss/index.scss [hunk 7]
```diff
@@ -1 +1,131 @@
...
+.followers-list {
+   
+    li {
+        list-style: none;
+
+        .follower-image-container {
+            width: 50px;
+            height: 50px;
+            border-radius: 50%;
+            background-color: #eab5c0;
```
Comment (Fermain) [inline src/scss/index.scss]: Variables? Is this colour part of your theme?

File: src/scss/index.scss [hunk 8]
```diff
@@ -1 +1,131 @@
...
+            display: flex;
```
Comment (Fermain) [inline src/scss/index.scss]: `d-flex`

File: src/scss/index.scss [hunk 9]
```diff
@@ -1 +1,131 @@
...
+            align-items: center;
```
Comment (Fermain) [inline src/scss/index.scss]: `align-items-center`

File: src/scss/index.scss [hunk 10]
```diff
@@ -1 +1,131 @@
...
+            justify-content: center;
```
Comment (Fermain) [inline src/scss/index.scss]: `justify-content-center`

File: src/scss/index.scss [hunk 11]
```diff
@@ -1 +1,131 @@
...
+        }
+
+        .follower-image-content {
+            display: flex;
```
Comment (Fermain) [inline src/scss/index.scss]: `d-flex`

File: src/scss/index.scss [hunk 12]
```diff
@@ -1 +1,131 @@
...
+            flex-direction: column;
```
Comment (Fermain) [inline src/scss/index.scss]: `flex-column`

File: src/scss/index.scss [hunk 13]
```diff
@@ -1 +1,131 @@
...
+          
+            p {
+                text-align: left;
```
Comment (Fermain) [inline src/scss/index.scss]: https://getbootstrap.com/docs/5.3/utilities/text/

**Review by Fermain — CHANGES_REQUESTED** [review]
Your HTML is good quality, but there are too many custom styles in SCSS that could be handled with Bootstrap classes.

File: src/scss/index.scss (hunk 13 above)
Comment (JeanArce) [reply src/scss/index.scss]: Thank you Oliver. i will fix it all.

**Pull request merged**